docs/static/releasenotes.asciidoc

```diff
@@ -5,9 +5,54 @@ coming[8.0.0]
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-0-0-beta1]]
+=== Logstash 8.0.0-beta1 Release Notes
+
+==== Breaking Changes
+
+[[java-11-minimum]]
```

Comment:
```suggestion
[[rn-java-11-minimum]]
```

Reply:
We can't reuse anchors, and I don't want to waste them on release notes.

Reply:
@karenzone this file has a few other anchors that aren't prefixed by `rn-`, should we change them as well?